guestbook.module

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/**
+* @file
+* Contains hooks for guestbook module.
+*/
+
+module_load_include('inc', 'guestbook', 'include/guestbook.form');
+module_load_include('inc', 'guestbook', 'include/guestbook.core');
+module_load_include('inc', 'guestbook', 'include/guestbook.theme');
+
+/**
+ * URL for downloading xml weather file.
+ */
+define('WEATHER_URL', 'https://xml.meteoservice.ru/export/gismeteo/point/2897.xml');
```

Comment:
@PavelSPN All constants inside module should be prefixed by module name like `GUESTBOOK_WEATHER_URL`